app/src/main/java/org/openobservatory/ooniprobe/activity/ResultActivity.java

```diff
@@ -0,0 +1,100 @@
+package org.openobservatory.ooniprobe.activity;
+
+import android.content.Intent;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.Menu;
+import org.openobservatory.ooniprobe.R;
+import org.openobservatory.ooniprobe.fragment.ResultFragment;
+import org.openobservatory.ooniprobe.fragment.ResultListFragment;
+import org.openobservatory.ooniprobe.utils.LogUtils;
+
+public class ResultActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_result);
+        Intent intent = getIntent();
+        if(intent.getExtras() != null) {
+            String json_file = intent.getStringExtra("json_file");
+            final String[] parts = LogUtils.getLogParts(this, json_file);
+            if (parts.length == 1){
+                Fragment fragment = new ResultFragment();
+                FragmentManager fm= getSupportFragmentManager();
+                FragmentTransaction ft=fm.beginTransaction();
+                Bundle bundle = new Bundle();
+                bundle.putInt("position", 0);
+                fragment.setArguments(bundle);
+                //ft.remove(fragment);
+                ft.add(R.id.fragment,fragment);
+                //ft.replace(R.id.fragment, fragment);
+                //ft.addToBackStack(null);
```

Comment:
Many tries to get it working. Will keep there until I don't test fragment with every android version.